# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../glibc/allow_Os.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://sourceware.org/bugzilla/show_bug.cgi?id=5203

objdir-x86_64-unknown-linux-gnu/libc_pic.os: In function `faccessat':
(.text+0x7fad9): undefined reference to `fstatat64'
collect2: ld returned 1 exit status

--- ./io/sys/stat.h.orig	2009-09-30 19:04:20.000000000 +0200
+++ ./io/sys/stat.h	2009-09-30 19:06:15.000000000 +0200
@@ -444,7 +444,11 @@
 		       __mode_t __mode, __dev_t *__dev)
      __THROW __nonnull ((3, 5));
 
-#if defined __GNUC__ && __GNUC__ >= 2 && defined __USE_EXTERN_INLINES
+#if defined __GNUC__ && __GNUC__ >= 2
+# ifndef __extern_inline
+#  define __extern_inline        extern inline
+# endif
+
 /* Inlined versions of the real stat and mknod functions.  */
 
 __extern_inline int
